app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryname/EntryNameContract.java

```diff
@@ -0,0 +1,23 @@
+package io.intrepid.contest.screens.entrysubmission.entryname;
+
+import io.intrepid.contest.base.BaseContract;
+
+public class EntryNameContract {
+    interface View extends BaseContract.View {
+        void onContestNameRetrieved(String contestName);
+
```

Comment:
We were thinking the same! But every time we run the Intrepid codestyle it changes the spaces back... did you have to modify your codestyle?